Documentation/network/servicemesh/mutual-authentication/mutual-authentication-example.rst

```diff
@@ -0,0 +1,235 @@
+
+There are as many entries as there are identities. Verify that these match by running the command:
+
+.. code-block:: shell-session
+    
+    kubectl get ciliumidentities
+
+The identify ID listed under ``NAME`` should match with the digits at the end of the SPIFFE ID executed in the previous command.
+
+
+Enforce Mutual Authentication
+=============================
+
+Rolling out mutual authentication with Cilium is as simple as adding the following to an existing or new CiliumNetworkPolicy.
+
+Add the following to your egress or ingress rules:
+
+.. code-block:: shell-session
+
+    authentication:
+        mode: "required"
+
+Update the existing rule to only allow ingress access to mutually authenticated workloads to access *echo* using:
+
+.. parsed-literal::
+
+    $ kubectl apply -f |SCM_WEB|\/examples/kubernetes/servicemesh/cnp-with-mutual-auth.yaml
+
+Verify Mutual Authentication
+===============================
+
+Re-try your connectivity tests. They should give similar results as before. Has mutual authentication actually happened?
+
+.. code-block:: shell-session
+
+    $ kubectl exec -it pod-worker -- curl -s -o /dev/null -w "%{http_code}" http://echo:8080/headers
+    200
+    $ kubectl exec -it pod-worker -- curl http://echo:8080/headers-1
+    Access denied
+
+Verify that mutual authentication has happened by accessing the logs on the agent. Start by enabling debug level (avoid doing this in production):
+
+.. code-block:: shell-session
+
+    cilium config set debug true
+
+Examine the logs on the Cilium agent located in the same node as the *echo* Pod. 
+For brevity, we search for some specific log messages:
+
+.. code-block:: shell-session
+
+    $ kubectl -n kube-system -c cilium-agent logs cilium-9pshw --timestamps=true | grep "Policy is requiring authentication\|Validating Server SNI\|Validated certificate\|Successfully authenticated"
+    2023-07-04T17:58:28.795760597Z level=debug msg="Policy is requiring authentication" auth_type=spire local_identity=17947 remote_identity=39239 subsys=auth
+    2023-07-04T17:58:28.800509503Z level=debug msg="Validating Server SNI" SNI ID=39239 subsys=auth
+    2023-07-04T17:58:28.800525190Z level=debug msg="Validated certificate" subsys=auth uri-san="[spiffe://spiffe.cilium/identity/39239]"
+    2023-07-04T17:58:28.801441968Z level=debug msg="Successfully authenticated" auth_type=spire local_identity=17947 remote_identity=39239 remote_node_ip=10.0.1.175 subsys=auth
+
+When you apply a mutual authentication policy, the agent retrieves the identity of the source Pod, connects to the node where the destination Pod is running and performs a TLS handshake.
+As the TLS handshake succeeded, then the connection was authenticated, the traffic protected by policy could proceed and packets between the two Pods can flow until the network policy is removed or the entry expires.
```

Comment:
Let's break this up in to two sentences:

```suggestion
As the TLS handshake succeeded, the connection was authenticated and the traffic protected by policy could proceed. Packets between the two Pods can flow until the network policy is removed or the entry expires.
```

Can you expand on what you mean by an entry expiring here? When does an entry expire?